CONTRIBUTING.md

```diff
@@ -18,8 +18,11 @@
 
 1. [Open an issue here](https://github.com/NuGet/Home/issues) and get some feedback from the NuGet team.
 1. Follow the instructions in [Code](#code)
-1. Make your change, and add tests.
+1. Make your change, and add tests. Please name your branch `dev-<userid>-<very-short-title>`.
 1. Create a [pull request](https://github.com/NuGet/NuGet.Client/pulls).
+    * Keep the pull request template, and link to an issue [NuGet/Home](https://github.com/NuGet/Home/issues). Create a new issue if you cannot find an existing one. Use a meaningful PR title, not the autogenerated title based on the branch name.
+    * All PRs created by someone outside the NuGet team will be assigned the `Community` label, and a team member will be assigned as the PR shepherd, who will be responsible for making sure the PR gets reviewed (even if they don't review it themselves), and periodically check the PR for progress.
```

Comment:
```suggestion
    * All PRs created by someone outside the NuGet team will be assigned the `Community` label, and a team member will be assigned as the PR shepherd, who will be responsible for making sure the PR gets reviewed (even if they don't review it themselves), and periodically check the PR for progress. Our CI will be run once the shepherd applies the `Approved for CI` label.
```

Reply:
Don't forget to add, "To invoke CI again, the label had to be reapplied, every single time!"